content/source/docs/enterprise/api/template.md

```diff
@@ -0,0 +1,130 @@
+Follow this template to format each API method. There are usually multiple sections like this on a given API endpoint page.
```

Comment:
I've left this as a raw .md file and not linked it in the sidebar, since it's only relevant when you're writing text in this directory. I think our users would consider this inside baseball.